### PR TITLE
Feature youtube clone

### PR DESCRIPTION
**Pull request by zubiix**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+              <svg class="video__item_icon" viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope yt-icon">
+                  <path d="M15 3H6c-.83 0-1.54.5-1.84 1.22l-3.02 7.05c-.09.23-.14.47-.14.73v1.91l.01.01L1 14c0 1.1.9 2 2 2h6.31l-.95 4.57-.03.32c0 .41.17.79.44 1.06L9.83 23l6.59-6.59c.36-.36.58-.86.58-1.41V5c0-1.1-.9-2-2-2zm4 0v12h4V3h-4z" class="style-scope yt-icon"></path>
+                </g></svg>
+              <span>443</span>
+          </div>
+          <div class="section__video_social_item">
+          <svg class="video__item_icon" viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false"  style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope yt-icon">
+              <path d="M11.7333 8.26667V4L19.2 11.4667L11.7333 18.9333V14.56C6.4 14.56 2.66667 16.2667 0 20C1.06667 14.6667 4.26667 9.33333 11.7333 8.26667Z" class="style-scope yt-icon"></path>
+            </g></svg>
+              <span>SHARE</span>
```
Comment (pelzolga123) [inline index.html]: 'Share' should be a button.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,430 @@
...
+  font-size: 11px;
+}
+
+.search svg path,
+.nav-buttons svg path {
+  fill: #8c8c8c;
+}
+
+.nav-buttons {
+  /* flex: 1; */
```
Comment (pelzolga123) [inline style.css]: Please remove comments.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,430 @@
...
+  justify-content: center;
+  margin: 0px 5%;
+}
+
+section {
+  display: flex;
+  padding: 24px 24px 0 0;
+  flex-direction: column;
+  flex: 3 2 0;
+  /* background: rebeccapurple; */
```
Comment (pelzolga123) [inline style.css]: Please remove comments.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,430 @@
...
+
+span.video--comment-action-count {
+  font-size: 13px;
+  margin-right: 8px;
+  color: #606060;
+}
+
+/* RECOMMENDATION SECTION */
+aside {
+  /* background: lime; */
```
Comment (pelzolga123) [inline style.css]: Please remove comments.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,430 @@
...
+  color: 606060;
+}
+.video--recommendation-autoplay > span {
+  font-size: 16px;
+  color: #0d0d0d;
+  flex: 1 0 auto;
+}
+
+li:first-child {
+  /* border: 1px solid black; */
```
Comment (pelzolga123) [inline style.css]: Please remove comments.

**Comment by pelzolga123** [comment]
## Hi Teaml!  
#### Your project looks :+1: amazing!
#### But you need to fix a few things to have this project approved:

* [ ] Share should be a button.
image
* [ ] Please delete all inline comments from css file.
image
* [ ] Readme file should be descriptive. A description of the project, a link to the live demo (using the http://raw.githack.com/), also you can add information about what you used for creating this project.
image

### Kindly make changes and submit another code review request.

If you need help or have some questions please find me on slack. *Happy coding*

**Comment by zubiix** [comment]
@pelzolga123 we have resolved the pending issues.

**Comment by pelzolga123** [comment]
## Great  work, Congratulation :+1: Project Approved.
You did all changes required previously. But check your HTML using a HTML validator: https://validator.w3.org. Fix the errors. I trust you to merge to master only after solving these minor errors. No need to submit for code review again. Good luck on the next project

**Pull request merged**